docker-compose.dev.yml

```diff
@@ -5,6 +5,10 @@ services:
       - "./nmdc_server:/app/nmdc_server"
     command: /start-reload.sh
 
+  db:
+    environment:
+      - POSTGRES_DB=nmdc_a
```

Comment:
Is this necessary? It's a departure from the conditions in production (no default database).  Could this cause something to work by accident in local dev that would break in prod?

Reply:
The reason I set this is because `nmdc_a` appears to be what's hardcoded everywhere else in the dev environment, but [these lines](https://github.com/microbiomedata/nmdc-server/blob/main/docker-compose.yml#L15-L19) would otherwise cause it to be set to `nmdc`, and I have no idea why.